src/mono/System.Private.CoreLib/src/System/Diagnostics/Tracing/NativeRuntimeEventSource.PortableThreadPool.NativeSinks.Mono.cs

```diff
@@ -68,5 +72,28 @@ internal sealed partial class NativeRuntimeEventSource : EventSource
             uint Count,
             ushort ClrInstanceID
         );
+
```

Comment:
Shouldn't this use the same mechanism used by the other events, calling an icall that in turn will use the FireEtw generated function in native code?

Reply:
Missed that similar question is already asked above.